packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -185,6 +185,42 @@ export class ProjectService {
         return [adapter, explores];
     }
 
+    async delete(projectUuid: string, user: SessionUser): Promise<void> {
+        if (user.ability.cannot('delete', 'Project')) {
+            throw new ForbiddenError();
+        }
+        // Check it is not the last project
+        /* const [admin, ...remainingAdmins] =
+        await this.organizationMemberProfileModel.getOrganizationAdmins(
+            user.organizationUuid,
+        );
+        if (
+            remainingAdmins.length === 0 &&
+            admin.userUuid === userUuidToDelete
+        ) {
+            throw new ForbiddenError(
+                'Organization must have at least one admin',
+            );
+        } */
```

Comment:
We can remove this now that we agree we want to allow users to delete all projects